tests/projects/test_projects.py

```diff
@@ -1,13 +1,16 @@
 import os
```

Comment:
Is there any way we can test the Databricks logic through mocks? It's getting somewhat complicated now.

Reply:
Yeah good call, will add some tests to this PR